test/regression/runSingleTest.ts

```diff
@@ -3,22 +3,88 @@ import {By, snap, Snappit, IConfig, WebDriver, WebElement} from "snappit-visual-
 
 import * as util from "../common/util";
 
-export async function regressionTest(browser: string, component: string) {
-    
+
+export async function regressionTest(browser: string, component: string, buildIdentifier: string) {
+    interface IResolutions {
+        [key:string]: string
+        firefox: string;
+        chrome: string;
+        safari: string;
+        "internet explorer": string;
+        MicrosoftEdge: string;
+    }
+
+    const screenResolution = "1600x1200";
+
+    interface IPlatforms {
+        [key: string]: string
+        firefox: string;
+        chrome: string;
+        safari: string;
+        "internet explorer": string;
+        MicrosoftEdge: string;
+    }
+
+    const MAC = "macOS 10.13";
+    const platforms: IPlatforms = {
+        firefox: MAC,
+        chrome: MAC,
+        safari: MAC,
+        "internet explorer": "Windows 10",
+        MicrosoftEdge: "Windows 10"
+    };
+    const platform = platforms[browser];
+
     const config: IConfig = {
-        browser: browser
+        browser: browser,
+        screenshotsDir: "artifacts/regressionScreenshots",
+        logException: [
+            "MISMATCH",
+            "NO_BASELINE",
+            "SIZE_DIFFERENCE",
+        ],
+        sauceLabs: {
+            platform: platform,
+            version: "latest",
+            screenResolution: screenResolution,
+            extendedDebugging: true,
+            maxDuration: 120,
```

Comment:
Do we need to comment that this value is in `seconds` and not `milliseconds`?

Reply:
The max duration value is in seconds. This configuration will ensure that the sauce labs test don't exceed that time limit. 

NOTE: As per Sauce Labs, best test scripts "A test should never last more than 30 minutes and ideally should take less than five minutes."

Reply:
@CITguy it's commented already in the upstream configuration documentation.

And by "commented', I mean....the way I do it. 😬 

https://github.com/rackerlabs/snappit-visual-regression/blob/db27eb891d7c0d38abf9190d36cfcc2ad231e1fe/src/config.ts#L31